media/css/mozorg/home/home-quantum.scss

```diff
@@ -0,0 +1,742 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import '../../pebbles/includes/lib';
+@import '../../hubs/buttons';
+@import '../../hubs/masthead';
+@import '../../quantum/buttons';
+
+// offset of nav logo at 1000px+ viewports
+$nav-margin: 84px;
+
+body {
+    border-top: 0;
+}
+
+// hide download button in the nav
+#global-nav-download-firefox {
+    display: none;
+}
+
+// adjust nav spacing to use space freed up by hiding the dl button
+.moz-global-nav .nav-primary-links {
+    @media #{$mq-tablet} {
+        width: calc(100% - 110px);
+    }
+
+    @media #{$mq-desktop} {
+        width: calc(100% - 220px);
+    }
+}
+
+main {
+    overflow: hidden;
+}
+
+.intro-section {
+    text-align: center;
+}
+
+#quantum {
+    @include open-sans;
+    background: #0060df;
+    background: url('/media/img/firefox/quantum/fox-tail-header.png') center center no-repeat,
+                linear-gradient(to bottom, #0060df 7%, #0095ff 100%);
+    color: #fff;
+    padding: 40px 0;
+
+    h2 {
+        @include font-size-level2;
+        margin-bottom: 1em;
+    }
+
+    p {
+        @include font-size-level5;
+        font-weight: normal;
+        line-height: 1.5;
+        margin: 0 auto 1em;
```

Comment:
@jpetto Sorry, just spotted a small bug I missed earlier. This should switch to `margin: 0 0 1em;` at tablet width, otherwise it ends up centered when it hits its max-width when everything else is left-aligned. I'd open a PR myself but I know we'll be making a few more home updates on Monday so just making a note here.